crates/cli/src/main.rs

```diff
@@ -101,7 +101,7 @@ async fn main() -> MainResult {
             Commands::Uninstall(args) => commands::uninstall(session, args).await,
             Commands::Unpin(args) => commands::unpin(session, args).await,
             Commands::Upgrade => commands::upgrade(session).await,
-            Commands::Use => commands::install_all(session).await,
+            Commands::Use => commands::install_all(session, args).await,
```

Comment:
You'll need to update the `Use` variant also.

Reply:
Ok